monkey/agent_plugins/credentials_collectors/chrome/src/decrypt.py

```diff
@@ -0,0 +1,39 @@
+# TODO: See if another module can replace this one
+from Crypto.Cipher import AES
+
+from .aes import AESModeOfOperationCBC
+
+
+def decrypt_AES(
+    encrypted_value: bytes, decryption_key: bytes, init_vector: bytes, block_size: int
+) -> str:
+    encrypted_value = encrypted_value[3:]
+    aes = AESModeOfOperationCBC(decryption_key, iv=init_vector)
+    cleartxt = b"".join(
+        [
+            aes.decrypt(encrypted_value[i : i + block_size])
+            for i in range(0, len(encrypted_value), block_size)
+        ]
+    )
+    return _remove_padding(cleartxt).decode()
+
+
+def _remove_padding(data: bytes) -> bytes:
+    """
+    Remove PKCS#7 padding
+    """
+    nb = data[-1]
+
+    try:
+        return data[:-nb]
+    except Exception:
+        return data
+
+
+def decrypt_v80(buff, master_key) -> str:
```

Comment:
Is this code duplicated in the Windows processor?

Reply:
Yes, both have this function

Reply:
Can we reduce duplication?

Reply:
Yes, I added a task to the issue so that we don't forget. The Windows implementation should use this implementation